minpy/core.py

```diff
@@ -15,16 +15,19 @@
 _logger = log.get_logger(__name__)
 
 
-def grad_and_loss(func, argnum=0):
+def grad_and_loss(func, argnum=0, method=False):
```

Comment:
Examples? Where is the use case for this support? For a method, I could always use `argnum` to specify which gradient I want to compute ? not?